maxrefdes178-FaceId/maxrefdes178_max32666/Makefile

```diff
@@ -43,7 +43,7 @@ TARGET=MAX32665
 endif
 
 # TODO remove this line
-BUILD_DIR=/build
+# BUILD_DIR=/build
```

Comment:
changing make fail to build in your workspace may result in failure depending on the path length, and probably fails in github action build.